Makefile

```
@@ -25,13 +25,10 @@ isort:
 isort-check:
 	python -m isort --check $(APP_LIST)
 
```

```suggestion

migrations-check:
	python -m manage makemigrations --check --dry-run
```
I would leave this method in the Makefile, so that you can run it also without running the test.